server/utils/constants.js

```diff
@@ -762,6 +762,13 @@ const DEVICE_POLL_FREQUENCIES = {
   EVERY_SECONDS: 1 * 1000,
 };
 
+const DEVICE_ROTATION = {
+  DEGRES_0: 0,
```

Comment:
*DEGREES in english :) 

https://context.reverso.net/traduction/anglais-francais/degrees+fahrenheit

Reply:
oops